addons/website_blog/views/snippets/s_blog_posts.xml

```diff
@@ -4,103 +4,142 @@
         <t t-set="record" t-value="data['_record']"/>
-        <div class="card">
-            <a class="s_blog_posts_post_cover" t-att-href="data['call_to_action_url']">
-                <t t-call="website.record_cover">
-                    <t t-set="_record" t-value="record"/>
-                    <t t-set="_resize_height" t-value="512"/>
-                    <t t-set="_resize_width" t-value="512"/>
-                    <t t-set="additionnal_classes" t-value="'thumb'"/>
-                </t>
+        <div class="card h-100">
+            <a class="text-decoration-none text-reset" t-att-href="data['call_to_action_url']" t-att-title="'Read ' + data['name']">
+                <div class="s_blog_posts_post_cover o_not_editable card-img-top ratio ratio-16x9">
+                    <t t-call="website.record_cover">
+                        <t t-set="_record" t-value="record"/>
+                        <t t-set="additionnal_classes" t-value="'position-absolute w-100 h-100 bg-600'"/>
+                    </t>
+                </div>
+                <div class="card-body">
+                    <div t-if="is_sample" class="h5 o_ribbon_right bg-primary text-uppercase">Sample</div>
+                    <h5 class="text-truncate" t-field="record.name"/>
+                    <t t-call="website_blog.s_dynamic_snippet_template_date_placeholder"/>
+                    <t t-call="website_blog.s_dynamic_snippet_template_teaser_placeholder"/>
+                </div>
             </a>
-            <div class="card-body">
-                <div t-if="is_sample" class="h5 o_ribbon_right bg-primary text-uppercase">Sample</div>
-                <a t-att-href="data['call_to_action_url']"><h4 class="mb-0" t-field="record.name"/></a>
-            </div>
-            <div class="card-footer d-flex justify-content-between">
-                <span class="text-muted mb-0" t-field="record.post_date" t-options='{"format": "MMM d, yyyy"}' />
-                <span class="text-muted mb-0">In <a class="fw-bold" t-field="record.blog_id.name" t-att-href="'/blog/%s' % record.blog_id.id" />
+            <div class="card-footer d-flex justify-content-between mt-auto border-0 p-3 pt-0 bg-transparent small">
+                <t t-call="website_blog.s_dynamic_snippet_template_author_placeholder"/>
+                <span class="text-muted">In <a class="fw-bold" t-field="record.blog_id.name" t-att-href="'/blog/%s' % record.blog_id.id" />
                     <a t-if="is_sample" class="fw-bold" href="#">Sample</a>
                 </span>
             </div>
         </div>
     </div>
 </template>
 
+<!-- Author ON / OFF -->
+<template id="website_blog.s_dynamic_snippet_template_author_placeholder" name="Author placeholder"/>
+<template id="website_blog.s_dynamic_snippet_template_author" inherit_id="website_blog.s_dynamic_snippet_template_author_placeholder" name="Author" active="True">
```

Comment:
@anso-odoo Ok this is what you were explaining to me :) I think this is weird: do you agree or is this what you wanted? If this is not what you wanted, I can assign a dev to have a look at how it should be implemented (after you process the other comments :pray:).

Reply:
Hello @qsm-odoo, thanks a lot for the review !

I agree that it's weird in a functional point of view. (For ref.: At this state of the PR, hiding/displaying these pieces of information has an impact on all the instances of `s_blog_post_snippet`s used across the website -> It shouldn't be the case.)

On a technical point of view, I did it this way this because in my mind it was cleaner than using something like `data-select-class="s_blog_post_author_hidden"` combined SCSS rules to `display: none` the information.

Anyway, I think it would be great if a dev could have a look indeed. I would be really happy to know the best solution for this !

Thanks !